src/msw/textctrl.cpp

```diff
@@ -1198,6 +1198,56 @@ void wxTextCtrl::SetRTFValue(const wxString& val)
     SetInsertionPoint(0);
 }
 
+wxTextSearchResult wxTextCtrl::SearchText(const wxTextSearch& search) const
+{
+    // set up the flags
+    WPARAM flags = 0;
+    if (search.m_direction == wxTextSearchDirection::Down)
+    {
+        flags |= FR_DOWN;
+    }
```

Comment:
One of big benefits of using an `enum` rather than just a `bool` is to ensure that we handle all the case, so could we please write this like

```suggestion
    switch ( search.m_direction )
    {
        case wxTextSearchDirection::Down:
            flags |= FR_DOWN;
            break;
            
        case wxTextSearchDirection::Up:
            // Nothing to do this is (surprisingly) the default.
            break;
    }
```

instead please?